docs/node/workflows.md

```diff
@@ -0,0 +1,161 @@
+---
+id: workflows
+title: Workflows in Node.js
+sidebar_label: Workflows
+description: In the Temporal Node SDK programming model, a Workflow is an exportable function that adheres to a set of rules.
+image: /img/workflow.png
+---
+
+[API reference](https://nodejs.temporal.io/api/modules/workflow)
+
+Workflows are the core of the Temporal system, they abstract away the complexities of writing distributed programs.
+
+In the NodeJS SDK, each Workflow runs in a separate V8 isolate to provide a [deterministic runtime](/docs/node/determinism).
+
+## How to write Workflow code
```

Comment:
In the near future, we should move pieces like this into the content directory -